src/ShopifyApp/Traits/AuthController.php

```diff
@@ -27,12 +27,13 @@ trait AuthController
     public function authenticate(Request $request, AuthenticateShop $authShop)
     {
         // Get the shop domain
-        if (Util::getShopifyConfig('turbo_enabled') && $request->user()) {
-            // If the user clicked on any link before load Turbo and receiving the token
-            $shopDomain = $request->user()->getDomain();
+        $shopDomain = $request->has('shop')
```

Comment:
this doesn't feel very safe

also where did the check if turbo is even enabled go?